src/main/reactjs/src/components/clerkTranslator/ClerkTranslatorListing.tsx

```diff
@@ -71,11 +71,9 @@ const ListingRow = ({
       </TableCell>
       <TableCell>
         <div className="rows">
-          {authorisations.map(({ langPair }, idx) => (
+          {authorisations.map(({ languagePair: lp }, idx) => (
```

Comment:
Voisi tässä mennä pidemällekin ja destrukturoida suoraan tarvittavat osat: `{ languagePair: { from, to } }`